SPECS-EXTENDED/azl-otel-collector/azl-otel-collector.spec

```diff
@@ -0,0 +1,63 @@
+%global debug_package   %{nil}
+
+Summary:        Azure Linux OpenTelemetry Collector Distribution
+Name:           azl-otel-collector
+Version:        0.123.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/microsoft/azl-otel-collector
+Source0:        https://azurelinuxsrcstorage.blob.core.windows.net/sources/core/azl-otel-collector-0.123.0-3.tar.gz
+Source1:        %{name}-%{version}-vendor-1.tar.gz
```

Comment:
what is in this file?

Reply:
the sources will be the azl-otel-collector and the vendor dependencies